storage/src/main/java/tech/pegasys/artemis/storage/server/rocksdb/dataaccess/V3RocksDbDao.java

```diff
@@ -324,6 +330,16 @@ public void addHotStates(final Map<Bytes32, BeaconState> states) {
       states.forEach(this::addHotState);
     }
 
+    @Override
+    public void addVote(final UnsignedLong validatorIndex, final VoteTracker vote) {
+      transaction.put(V3Schema.VOTES, validatorIndex, vote);
+    }
```

Comment:
Do we actually need to add a single vote or is `addVotes` all we'll actually use?

Reply:
`addVotes` is all we'll actually use, so I might remove that.

Reply:
done.